R/dfm-classes.R

```diff
@@ -102,12 +102,12 @@ setMethod("rowMeans",
 #' @rdname dfm-class
 setMethod("+", signature(e1 = "dfm", e2 = "numeric"),
           function(e1, e2) {
-              as.dfm(as(e1, "dgCMatrix") + e2)
+            reassign_slots(as.dfm(as(e1, "dgCMatrix") + e2), e1)
```

Comment:
@koheiw you added all of the `as.dfm()` functions to ensure backwards compatibility with older dfm versions that lacked slots we may have added later. Can you check whether this is the most stable way to keep the slot? It seems more consistent to use the `reassign_slots()` in `as.dfm()` to save the existing slot values there, rather than in each dfm-modifying function.